website/content/docs/sync/github.mdx

```diff
@@ -51,6 +51,30 @@ If you decide to go with an access token, here's how you can configure a sync de
 
   </CodeBlockConfig>
 
+  Configure an environment sync destination:
+
+  ```shell-session
+  $ vault write sys/sync/destinations/gh/my-dest \
+      access_token="$ACCESS_TOKEN" \
+      repository_owner="$OWNER" \
+      repository_name="$NAME"
+      environment_name="$ENVIRONMENT"
+  ```
+
+  **Output:**
+
+  <CodeBlockConfig hideClipboard>
+
+  ```plaintext
+  Key                   Value
+  ---                   -----
+  connection_details    map[access_token:***** environment_name:<environment> repository_owner:<owner> repository_name:<name>]
+  name                  my-dest
+  type                  gh
+  ```
+
+  </CodeBlockConfig>
```

Comment:
````suggestion
  Use `vault write` to configure an environment sync destination:

  ```shell-session
  $ vault write sys/sync/destinations/gh/DESTINATION_NAME \
      access_token="GITHUB_ACCESS_TOKEN"                  \
      repository_owner="GITHUB_OWNER_NAME"                \
      repository_name="GITHUB_REPO_NAME"                  \
      environment_name="GITHUB_ENVIRONMENT_NAME"
  ```
  
  For example:

  <CodeBlockConfig hideClipboard>
  
  ```
  $ vault write sys/sync/destinations/gh/hcrepo-sandbox       \
      access_token="0000000000000000000000000000000000000000" \
      repository_owner="hashicorp"                            \
      repository_name="hcrepo"                                \
      environment_name="sandbox"

  Key                   Value
  ---                   -----
  connection_details    map[access_token:***** environment_name:sandbox repository_owner:hashicorp repository_name:hcrepo]
  name                  hcrepo-sandbox
  type                  gh
  ```

  </CodeBlockConfig>

````

Unless it's required or best practice to use environment variables for a particular parameter (like `${VAULT_ADDR}), we should use placeholders for the parameter values and then (if it makes sense) provide an actual example command with realistic values.